src/Common/ExecCommand.php

```diff
@@ -233,13 +254,15 @@ protected function executeCommand($command)
             $process->setWorkingDirectory($this->workingDirectory);
         }
         $this->getExecTimer()->start();
-        if ($this->isPrinted) {
-            $process->run(function ($type, $buffer) {
-                print $buffer;
-            });
-        } else {
-            $process->run();
-        }
+        $process->run(
+            function ($type, $buffer) {
+                if ($this->isOutputLogged) {
+                    $this->printTaskInfo($buffer);
```

Comment:
If we print it, we can't control the log level at all. It's all or nothing.

Using logOutput() allows output to be printed conditionally, depending on the verbosity level.